src/utils/ga.ts

```diff
@@ -0,0 +1,25 @@
+import { event, initialize, pageview } from "react-ga";
+import { REACT_APP_GA_MEASUREMENT_ID } from "secrets";
+
+export const initGoogleAnalytics = () => {
+  if (REACT_APP_GA_MEASUREMENT_ID) {
+    initialize(REACT_APP_GA_MEASUREMENT_ID, {
+      debug: true,
+    });
+    pageview(window.location.pathname + window.location.search);
```

Comment:
Is this going to work for all of the browsers? You need to test this on Safari and Mozilla, maybe Opera as well.

Reply:
https://support.google.com/analytics/answer/3541880?hl=en

Checked Firefox and Opera.

Reply:
I'm not talking about the google analytics, I'm talking about the `window` property and it's fields. Sometimes they are different on cross browsers.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Window#browser_compatibility
It also supported.